CHANGELOG.md

```diff
@@ -35,6 +63,21 @@ to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ### New features
 
+* The new `jj git colocation` command provides sub-commands to show the
+  colocation state (`status`) and to convert a non-colocated git repo into
```

Comment:
I think a comma instead of "and" on this line and a comma before the next "and" is slightly better grammar.